Signum.React.Extensions/Dynamic/View/Nodes.tsx

```diff
@@ -1276,6 +1279,13 @@ NodeUtils.register<SearchControlNode>({
     </FetchQueryDescription>
     <ExpressionOrValueComponent dn={dn} binding={Binding.create(dn.node, f => f.searchOnLoad)} type="boolean" defaultValue={null} />
     <ExpressionOrValueComponent dn={dn} binding={Binding.create(dn.node, f => f.showContextMenu)} type={null} defaultValue={null} exampleExpression={"fop => \"Basic\""} />
+    <ExpressionOrValueComponent dn={dn} binding={Binding.create(dn.node, f => f.extraButtons)} type={null} defaultValue={null} exampleExpression={`sc => [
+  { 
+    order: -1.1,
+    button: modules.React.createElement("button", { className: "btn btn-light", title: "Setting", onClick: e => alert(e) },
+                                          [modules.React.createElement(modules.FontAwesomeIcon, { icon: "cog", color: "green" }), " Setting"])

```

Comment:
You don't need the array, React.createElement accepts multiple arguments, otherwise you have to use key

Reply:
Ok, I will change and push tomorrow

Reply:
I have fixed and pushed.